### ISSUE TITLE
Accessibility:List: List item names are read twice when accessed

### ISSUE THREAD

**Issue by puru-hk** [SAP/fundamental-ngx#3476]
Description: List item names are read twice when accessed

Expected: Each & every UI should be read out only once when accessed

Screenshot: 

![image](https://user-images.githubusercontent.com/32538291/94647489-29c2f480-030e-11eb-9a3b-20f6a0116356.png)

![image](https://user-images.githubusercontent.com/32538291/94647493-2cbde500-030e-11eb-8033-3f4f2bee0e69.png)

**Comment by sKudum** [comment]
Duplicate  of #3834.Hence closing it.
